modules/rgbd/src/tsdf_functions.cpp

```diff
@@ -373,5 +373,90 @@ void integrateVolumeUnit(
 
 }
 
+/// Custom Volume Hash Table
+
+VolumesTable::VolumesTable() : bufferNums(1)
```

Comment:
TODO: move methods of this structure to header file

Reply:
done